### DISCUSSION TITLE
Develop windows desktop apps using mac ecosystem

### DISCUSSION THREAD

**Discussion by MrHazimAli** [microsoft/react-native-windows#9702] [Q&A] [answered]
Hye all, I just exploring to build a windows desktop app and found react-native-windows. I could not find a direct answer to my question which is.

Can we develop a Windows Desktop Application using macbook? 
Development - Macbook
Production product will be using windows

as I found that the step for basic windows need to run some script in powershell

Thanks!

**Comment by chrisglein** [comment, marked as answer]
[This article](https://microsoft.github.io/react-native-windows/docs/win10-vm) should hopefully have the answers you're looking for.
